src/network/core/config.h

```diff
@@ -91,4 +91,6 @@ static const uint NETWORK_GRF_NAME_LENGTH           =   80;           ///< Maxim
  */
 static const uint NETWORK_MAX_GRF_COUNT             =   255;
 
+static const uint NETWORK_SECRET_KEY_LENGTH = 32 * 2 + 1; ///< The maximum length of the hexadecimal encoded secret keys, in bytes including '\0'.
```

Comment:
Should this be based on a value from `network_crypto.h`? On one hand, it is nice that this doesn't need any other header file, on the other hand, they do strongly correlate. Maybe add a reference to it?

Reply:
They do. I added a comment, and added a `static_assert` in the test case.